chart/helm-operator/values.yaml

```diff
@@ -184,5 +184,5 @@ extraVolumeMounts: []
 extraVolumes: []
 resources:
   requests:
-    cpu: 50m
-    memory: 64Mi
+    cpu: 200m
```

Comment:
We landed on 150m to suit our clusters, however it is hard to give a suits everyone default. I can see the bumping the default some might make sense, @hiddeco thoughts?

Reply:
I have been quiet on this PR because I needed to confirm some statistics from various clusters, I will get back to this as soon as I have time, but other things have my priority at the moment.

Feel free to comment current resource usage in combination with the amount of releases and the enabled Helm versions, that will definitely help us get to the right amount of resources.